.github/workflows/Bundle.yml

```
@@ -0,0 +1,80 @@
+        run: git clone --branch 2.1.1 --recurse-submodules https://github.com/raspberrypi/pico-sdk.git
+
+      - name: Create Build Folder
+        run: |
+          mkdir build
+          mkdir output
+
+      - name: Build PicoStation Pico1
+        run: |
+          cd ${{ github.workspace }}/build
+          rm -fr ./*
+          cmake -DPICO_SDK_PATH=${{ github.workspace }}/pico-sdk -DPICOSTATION_VARIANT=picostation_pico1 .. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)